### PR TITLE
Update kaptain 2.0 version policy

### PR DESCRIPTION
**Pull request by juliangieseke**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-87586

## Description of changes being made

Updated version policy to reflect Kaptain 2.0 helm and DKP based deployment. I've basicly made this up thinking its the best change to be done, but not sure if we need some other people to have a look? support matrix is important I guess..

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4377.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kaptain/2.0.0/version-policy/index.md [hunk 1]
```diff
@@ -34,22 +29,21 @@ Advice on preventing and recovering from failures and troubleshooting, as availa
...
 Istio usage is only supported in conjunction with Kaptain and the configurations delivered as part of Kaptain.
 All other usage of [experimental]Istio[/experimental] is considered experimental.
 
-## Support for KUDO - Cassandra, Kafka & Spark
-### Support Definition - Secondary Support
-Secondary support covers support for the base technology of platform service, which is Cassandra, Kafka, and Spark, and additionally support for the KUDO-based operator of the mentioned platform services.
+## Secondary Support for Cassandra, Kafka & Spark
 
-*Base Technology refers to Cassandra, Kafka, and Spark.
+<p class="message--note"><strong>Secondary Support Definition:</strong> Secondary support covers support for the base technology of platform service, which is Cassandra, Kafka, and Spark.</p>
```
Comment (juliangieseke) [inline pages/dkp/kaptain/2.0.0/version-policy/index.md:39]: is this the correct html for this? I think it should be like "info" not red like this seems to be?

File: pages/dkp/kaptain/2.0.0/version-policy/index.md [hunk 2]
```diff
@@ -9,21 +9,16 @@ enterprise: false
...
+## Supported DKP Versions
 
-Kaptain 1.3.0 runs on DKP 2.1.1, Konvoy 1.7 and 1.8.
-
-## Supported Kommander Versions
-
-Kaptain 1.3.0 cannot be installed from Kommander or as an add on.
-
-Note that it is possible to use [Kommander's Dex instance](../configuration/external-dex) for authenticating users in Kaptain.
+Kaptain 2.0.0 runs on DKP 2.1.2 and newer and can be installed as a workspace catalog application.
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.0.0/version-policy/index.md:18]:
```suggestion
You can deploy Kaptain 2.0.0 on DKP 2.1.2 and later versions as a workspace catalog application.
```

File: pages/dkp/kaptain/2.0.0/version-policy/index.md (hunk 1 above)
Comment (gvasquezvargas) [reply pages/dkp/kaptain/2.0.0/version-policy/index.md:39]: That would be:
```suggestion
<p class="message--note"><strong>NOTE: </strong><strong>Secondary Support Definition:</strong> Secondary support covers support for the base technology of platform service, which is Cassandra, Kafka, and Spark.</p>
```

Comment (gvasquezvargas) [reply pages/dkp/kaptain/2.0.0/version-policy/index.md:39]: Looks good to me in front-end. What do you mean with red?

**Review by gvasquezvargas — APPROVED** [review]

File: pages/dkp/kaptain/2.0.0/version-policy/index.md (hunk 1 above)
Comment (juliangieseke) [reply pages/dkp/kaptain/2.0.0/version-policy/index.md:39]: nothing.. the message is purple, not red 🙈

**Review by Colin1841 — APPROVED** [review]

**Pull request merged**